### ISSUE TITLE
color maps

### ISSUE THREAD

**Issue by mwcraig** [eteq/nb-astroimage-api#5]
I don't think the current API includes anything about color maps. An implementation has been added at https://github.com/astropy/astrowidgets/pull/66/files but something should be added here too.

**Comment by pllim** [comment]
I think the concern of my PRs being increasingly Ginga-centric is a valid one. However, setting colormap programmatically should be a common enough need that it can be done for multiple backends?

**Comment by mwcraig** [comment]
> However, setting colormap programmatically should be a common enough need that it can be done for multiple backends

I agree 100% @pllim. I meant that the API proposal here should have something about colormaps added it to it (and was a little surprised that was overlooked initially).

I think the way you implemented it (a colormap is a string) is the way to go for all of the backends.
